schema-derivatives-perpfutures.graphql

```diff
@@ -1440,32 +1508,37 @@ type Account @entity{
 
   " All collateral taken out from trading events occurred in this account "
   collateralOut: [CollateralOut!]! @derivedFrom(field: "account")
-
   " All Position events occurred in this account "
   positions: [Position!]! @derivedFrom(field: "account")
 
   " All swaps events occurred in this account "
   swaps: [Swap!]! @derivedFrom(field: "account")
-
 }
 
-
 # Helper entity for calculating daily/hourly active users
-type ActiveAccount @entity (immutable:true) {
+type ActiveAccount @entity(immutable: true) {
   " { daily/hourly }-{ Address of the account }-{ Days/hours since Unix epoch } "
-  id: Bytes!
+  id: String!
+}
+
+enum PositionSide {
+  " Position opened as long "
+  LONG
+
+  " Position opened as short "
+  SHORT
 }
 
 # A position is defined as who has what position in the protocol and distribution of longs and shorts at a positional level
 type Position @entity {
   " { Account address }-{ Market address }-{ Position Side }-{ Optional: Interest Rate Type}-{ Counter } "
-  id: Bytes!
+  id: String!
```

Comment:
Keep IDs as `Bytes`. If we need to concat various values to get the ID you can still do so with Bytes. We don’t need the dashes, so it is all addresses and numbers. (Interes Rate Type doesn’t apply in this case, I think we can also remove from the comment)

Reply:
fixed at 3864986b411c56a979d1bde345c6a0abe72c0402 Thanks!